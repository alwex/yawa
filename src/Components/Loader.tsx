import React from 'react'
import { ActivityIndicator } from 'react-native'
import { secondaryColor } from '../Theme/Variables'
import { View } from 'native-base'
import style from './Styles/LoaderStyle'

export default class Loader extends React.Component {
  render() {
    return (
      <View style={style.container}>
        <ActivityIndicator size="large" color={secondaryColor} />
      </View>
    )
  }
}
