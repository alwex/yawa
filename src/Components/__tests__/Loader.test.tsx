import React from 'react'
import renderer from 'react-test-renderer'
import Loader from '../Loader'

describe('Loader', () => {
  it('render correctly', () => {
    const component = renderer.create(<Loader />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
