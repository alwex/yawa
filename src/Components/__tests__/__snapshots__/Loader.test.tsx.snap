// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader render correctly 1`] = `
<View
  style={
    Array [
      Object {},
      Object {
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#90C7D4"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
